### DISCUSSION TITLE
Unable to modify the firewall setting after running the script

### DISCUSSION THREAD

**Discussion by ghost** [0x6d69636b/windows_hardening#144] [Q&A]
After running the (finding_list_0x6d69636b_machine.csv) hardeningkitty script in my notebook, I cannot change any security setting unless reinstalling my windows 10 pro which doesn't belongs to any domain. 

Here is the message about the security setting. I am the only Admin user in this computer. 
![image](https://user-images.githubusercontent.com/102276303/219849738-a58b39f8-e3aa-4499-8d10-e6b299025bdd.png)

I have tried to delete the group policy and restore the computer to the restored point but the result is the same as above.
Did anyone encounter the same issue before?

**Comment by 0x6d69636b** [comment]
This is the behaviour if group policy settings are in use. Since you own a Windows 10 Pro, you can open the local group policy editor (gpedit.msc) and configure youre firewall settings there. HardeningKitty writes only registry keys, so you won't see the changes there. However, you can overwrite them there. If you don't want use group policy settings for your firewall, after you run HardeningKitty, you can delete all registry keys in the list with the category firewall and "Policy" at the end.

**Reply by ghost** [reply]
Thank you for your reply. Really Appreciate that. Do you mind telling me more about how to disable the group policy in order to solve the issue?  Thank you.

**Reply by 0x6d69636b** [reply]
Go to https://github.com/0x6d69636b/windows_hardening/blob/master/lists/finding_list_0x6d69636b_machine.csv and select all policies, like ID 1400 and open the registry path (regedit.exe, for ID 1400 it is _HKLM:\SOFTWARE\Policies\Microsoft\WindowsFirewall\DomainProfile_ -> in regedit it is _Computer\HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Microsoft\WindowsFirewall_). Select the registry item (for ID 1400 it is _EnableFirewall_) and delete the key.

![image](https://user-images.githubusercontent.com/13913706/219851391-48f41964-3064-4c36-a6e6-0325b6b84405.png)
